### DISCUSSION TITLE
Allow text formatting override on textarea Grid columns

### DISCUSSION THREAD

**Discussion by mcfarlandonline** [ExpressionEngine/ExpressionEngine#3001] [Feature Requests]
Textarea fieldtypes have an **Allow override?** option – to allow authors to override the default text formatting for the field from the publish form. However, textarea fields _within_ a grid fieldtype do not have this option.

**Feature request:** Allow text formatting override on textarea Grid columns. 

This would be helpful if, for example, the contained textarea fields is usually used for text and paragraphs, but occasionally needs to accommodate iframes/scripts for a video or other embeddable features.

**Comment by intoeetive** [comment]
@robinsowell I see in the code that this feature has been specifically disabled for textareas in Grid. Do you have any idea why that was done? That was back in the days, so might be not relevant anymore?

**Comment by robinsowell** [comment]
I vaguely recall there was brief discussion, but I won't swear to the details.  If I had to posit a reason, I'd say largely a design issue.  It's not a high demand option and when put into a grid, things really start getting tight.

But I don't believe there was any 'we don't want to allow this for reasons' sort of reason.

**Comment by intoeetive** [comment]
I have investigated this and appears that unlike fields, which have `field_ft_X` column in database to store formatting preference, Grid columns only have `col_id_X` and no formatting column. So adding this would be not just UI change, but an update to how data are stored
